uint8 command
---
bool result
